platform/broadcom/saibcm-modules/systems/linux/kernel/modules/include/uapi/linux/psample.h

```diff
@@ -0,0 +1,35 @@
+#ifndef __UAPI_PSAMPLE_H
+#define __UAPI_PSAMPLE_H
```

Comment:
This looks odd, I don't think this file should be used instead of what the system provides. @adyeung would you be able to comment on if this file needs to be present here for the kernel module build from SAI 8.4?

Reply:
Hi @saiarcot895,  These kernel headers were removed in PR #11709.   Use the system psample headers and keep the "psample_sample_packet" call changes from PR #11709.   These system header files were also removed in favor of kernel headers in a later version of SAI/SDK.

Reply:
@Gfrom2016 just to make sure this is clear, #11709 from Michael needs to take precedence over this submission, pls do the needful to honor #11709, otherwise the build will break.

Reply:
@saiarcot895 @michaelli10 and @adyeung, thanks for review this, I pushed another commit to honor #11709, please help review again, thanks